src/ui/style/palette.scss

```diff
@@ -65,6 +67,12 @@ $text-light: $primary-variant-1;
 
 // gradients
 $gradient: linear-gradient(270deg, #00b6b7 17.51%, #004073 100%);
+$card-background-light-gradient: linear-gradient(180deg, #f2f6f9 0%, rgba(242, 246, 249, 0) 100%);
+$card-background-dark-gradient: linear-gradient(
+  180deg,
+  rgba(36, 36, 51, 0.8) 0%,
+  rgba(36, 36, 51, 0) 100%
```

Comment:
These rgba colors corresponds to the hexa color `#242433` for [$card-dark](https://github.com/okp4/dataverse-portal/pull/301/files#diff-de1d1d6830d571cc32971e3256d2cbc067cde6a9285b4b22757f92c916102a08L52).
It's probably possible to update it using it's RGB value and reuse it?

Reply:
As discussed together, the issue of accessibility needs to be discussed with the product to see if users can benefit from keyboard navigation, and if so, to define the styles to be used in advance.